pkgs/shells/fish/plugins/fish-you-should-use.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildFishPlugin,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+buildFishPlugin {
+  pname = "fish-you-should-use";
+  version = "unstable-2022-02-13";
+
+  src = fetchFromGitHub {
+    owner = "paysonwallach";
+    repo = "fish-you-should-use";
+    rev = "a332823512c0b51e71516ebb8341db0528c87926";
+    hash = "sha256-AzH1rZFqEH8sovZZfJykvsEmCedEZWigQFHWHl6/PdE=";
+  };
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  meta = {
+    description = "A fish plugin that reminds you to use your aliases";
```

Description should not start with an indefinite article (See [meta.description](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes))

```suggestion
    description = "Fish plugin that reminds you to use your aliases";
```